subprojects/core/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequest.java

```diff
@@ -57,6 +57,17 @@ public String getVersion() {
         return version;
     }
 
+    @Nullable
+    @Override
+    public String getArtifact() {
+        return null;
+    }
+
+    @Override
+    public boolean isConfigured() {
```

Comment:
The idea here was we could show if when a plugin fails to apply that it was enriched (so the user may know that something changed)